java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Content Lifecycle sources will no longer include cloned channels by other projects (bsc#1137965)
```

Comment:
As per the message needs to be readable and clear to customer as well, I'd rephrase it a bit in a more generic way:

`Hide channels managed by Content Lifecycle projects from available sources (bsc#1137965)`